lib/main.dart

```diff
@@ -53,7 +52,8 @@ bool _isProcessRunning = false;
 const kLastBGTaskHeartBeatTime = "bg_task_hb_time";
 const kLastFGTaskHeartBeatTime = "fg_task_hb_time";
 const kHeartBeatFrequency = Duration(seconds: 1);
-const kFGSyncFrequency = Duration(minutes: 5);
+const kFGSyncFrequency =
```

Comment:
Can we undo this to avoid noise during normal development.

Reply:
Fixed.